.azure-pipelines/latest_metrics.yml

```
@@ -0,0 +1,24 @@
+schedules:
+ - cron: "0 23 * * 0"
+   displayName: Weekly CI (Sunday night UTC)
```

If you don't mind, we usually do https://github.com/DataDog/datadogpy/pull/509

```suggestion
- cron: "0 5 * * *"
  displayName: Nightly (EDT/EST)
```

---

That ends up being Sunday morning though - https://crontab.guru/#0_5_*_*_0

Wanted to avoid any issues popping up over the weekend for people and setting it for Sunday night meant that it would be done before Paris started checking things Monday morning.

What about Monday morning: https://crontab.guru/#0_5_*_*_1 ?

---

Edited!